source/_components/device_tracker.mqtt.markdown

```diff
@@ -26,15 +26,45 @@ device_tracker:
 
 {% configuration %}
 devices:
-  description: List of devices with their topic.
+  description: A dictionary where the keys are devices and the values are their corresponding topic.
   required: true
-  type: list
+  type: map
```

Comment:
I think this must be a list

Reply:
the example doesn't show it as a list, nor does the code:
`vol.Required(CONF_DEVICES): {cv.string: mqtt.valid_subscribe_topic}`

I can change both, but I was following the original convention

Reply:
Actually I looked at some examples and now I understand. Will change accordingly